modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language_fr.properties

```diff
@@ -72,12 +72,12 @@ privacy-policy=Politique de confidentialité
 privacy-policy-url=URL de la politique de confidentialité
 privacy-policy-url-help=Fournissez un lien vers la politique de confidentialité de votre application. Cette information est indiquée aux utilisateurs pendant le processus d'autorisation.
 privacy-policy-url-is-invalid=L'URL de la politique de confidentialité n'est pas valide.
-privacy-policy-url-scheme-is-invalid=La construction de l'URL de la politique de confidentialité n'est pas valide ; seules les constructions en http et https sont disponibles.
-redirect-uri-is-missing-for-grant-type-x=Le URI de rappel est manquant pour l'autorisation de type «{0} ».
+privacy-policy-url-scheme-is-invalid=La construction de l'URL de la politique de confidentialité n'est pas valide ; seules les constructions en http et https sont disponibles.
+redirect-uri-is-missing-for-grant-type-x=Le URI de rappel est manquant pour l'autorisation de type «{0} ».
 redirect-uri-x-fragment-is-invalid=Le URI de rappel «{0}» ne peut pas contenir un fragment.
 redirect-uri-x-is-invalid=Le URI de rappel «{0}» n'est pas valide
 redirect-uri-x-path-is-invalid=Le chemin du URI de rappel «{0}» n'est pas valide
-redirect-uri-x-scheme-is-invalid=La construction du URI de rappel "{0}" n'est pas valide ; seules les constructions en http et https sont disponibles.
+redirect-uri-x-scheme-is-invalid=La construction du URI de rappel "{0}" n'est pas valide ; seules les constructions en http et https sont disponibles.
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.